### ISSUE TITLE
Rename source directory

### ISSUE THREAD

**Issue by ajcse1** [jaws/jaws#75]
@czender In the current version of conda package, although it is downloaded named "jaws" in `miniconda/pkgs`, but it installs everything into a package named "source" in `miniconda/lib/python3.6/site-packages`. It takes the name of the directory in which python scripts are present. 
Currently, to access the installed package user has to do `import source` from python shell.
So, I think in order to be consistent, "source" directory should be renamed to "jaws", so that user can do `import jaws`
Should I make this change?

**Comment by czender** [comment]
Agreed. Definitely yes. Python usage should be "import jaws".

**Comment by ajcse1** [comment]
Done
